src/scss/components/dropdown-select/index.scss

```diff
@@ -73,6 +73,10 @@ hx-select-control {
   }
 }
 
+#selLongOption-disabled {
+  opacity: 1;
+}
+
```

Comment:
We should not need to add an `id` special case here.  Please update the  **`disabled`** mixins.

Reply:
Changes have been done, Please review